identifiers/country-in/vidhan_sabha.csv

```diff
@@ -768,4 +768,3800 @@ ocd-division/country:in/state:mh/district:yt/cd:pusad,Pusad constituency; Yavatm
+ocd-division/country:in/state:nl/district:ph/cd:chizami,Chizami  (ST) constituency;  Phek district; Nagaland
+ocd-division/country:in/state:nl/district:ph/cd:chozuba,Chozuba  (ST) constituency;  Phek district; Nagaland
+ocd-division/country:in/state:nl/district:ph/cd:phek,Phek  (ST) constituency;  Phek district; Nagaland
+ocd-division/country:in/state:nl/district:ph/cd:meluri,Meluri  (ST) constituency;  Phek district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:tuli,Tuli  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:arkakong,Arkakong  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:impur,Impur  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:angetyongpang,Angetyongpang  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:mongoya,Mongoya  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:aonglenden,Aonglenden  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:mokokchung_town,Mokokchung Town  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:koridang,Koridang  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:jangpetkong,Jangpetkong  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:mk/cd:alongtaki,Alongtaki  (ST) constituency;  Mokokchung district; Nagaland
+ocd-division/country:in/state:nl/district:zu/cd:akuluto,Akuluto  (ST) constituency;  Zunheboto district; Nagaland
+ocd-division/country:in/state:nl/district:zu/cd:atoizu,Atoizu  (ST) constituency;  Zunheboto district; Nagaland
+ocd-division/country:in/state:nl/district:zu/cd:suruhoto,Suruhoto  (ST) constituency;  Zunheboto district; Nagaland
+ocd-division/country:in/state:nl/district:zu/cd:aghunato,Aghunato  (ST) constituency;  Zunheboto district; Nagaland
+ocd-division/country:in/state:nl/district:zu/cd:zunheboto,Zunheboto  (ST) constituency;  Zunheboto district; Nagaland
+ocd-division/country:in/state:nl/district:zu/cd:satakha,Satakha  (ST) constituency;  Zunheboto district; Nagaland
+ocd-division/country:in/state:nl/district:wo/cd:tyui,Tyui  (ST) constituency;  Wokha district; Nagaland
+ocd-division/country:in/state:nl/district:wo/cd:wokha,Wokha  (ST) constituency;  Wokha district; Nagaland
+ocd-division/country:in/state:nl/district:wo/cd:sanis,Sanis  (ST) constituency;  Wokha district; Nagaland
+ocd-division/country:in/state:nl/district:wo/cd:bhandari,Bhandari  (ST) constituency;  Wokha district; Nagaland
+ocd-division/country:in/state:nl/district:mo/cd:tizit,Tizit  (ST) constituency;  Mon district; Nagaland
+ocd-division/country:in/state:nl/district:mo/cd:wakching,Wakching  (ST) con
```

Comment:
It looks like previously the districts were added in `federal_states_territories.csv`. I think these should be moved there as well.

Reply:
Done